src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -104,12 +119,13 @@ namespace OpenLoco::Ui::PromptBrowse
     static loco_global<char[32], 0x009D9E64> _filter;
     static loco_global<char[512], 0x009D9E84> _directory;
     static loco_global<char[512], 0x0112CC04> _stringFormatBuffer;
-    static loco_global<char[512], 0x011369A0> _text_input_buffer;
+    static std::string _textInputBuffer;
```

Comment:
`string` or `u8string`. As this is a interface function I can expect we will have to deal with utf8 encoding.

Reply:
Never mind will be for future work.